src/System.Private.Xml/src/System/Xml/Dom/XPathNodeList.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What's changing here and other places where the first line is showing as modified?  Is the file's BOM changing?